### PR TITLE
Do not add / for html pages in rewriteDocLinks

### PR DESCRIPTION
**Pull request by ougfh**
Signed-off-by: Anastasia Rodionova <argali96@gmail.com>

## Hey, I just made a Pull Request!

Fixes https://github.com/backstage/backstage/issues/5965

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by OrkoHunter — APPROVED** [review]
Makes sense

**Comment by OrkoHunter** [comment]
@backstage/maintainers Needs approval due to changeset 🙂

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**
